doc/subsystems/test/sanitycheck.rst

```diff
@@ -385,6 +385,64 @@ extra_sections: <list of extra binary sections>
     extra, unexpected sections in the Zephyr binary unless they are named
     here. They will not be included in the size calculation.
 
+harness: <list of harness>
+    A set of string harness for the testcase. This keyword means that a
+    special harness is needed to run the tests sucessfully. This can be
+    as simple as a loopback wiring or a complete hardware test setup for
+    sensor and IO testing. Usually pertains to external dependency domains
+    but can be anything such as console, sensor, net, keyboard, bluetooth.
```

Comment:
change to:  ``keyboard, or Bluetooth.``  (add the ``or`` and capitalize ``Bluetooth`` since it's a trademark)

Reply:
Thanks for figuring it out. Addressed.